### PR TITLE
Exe args

### LINKED ISSUE #30: Executable args
### Description
Our executables should take arguments so that the user can change certain default behavior:
* Client:
  * Server port
  * Server address
  * Filename
* Server
  * Listen port
  * Filename

Requires #26

### PR DESCRIPTION
**Pull request by BobbyBytes**
## Get executable args from command line input
* Closes #39 
Note :  Added the args for verbose output, to be implemented in #5  
# Client
- -f --filename 
- -p --port
- -a --address
- -v --verbose

# Server
- -f --filename 
- -p --port
- -v --verbose

### CODE REVIEW

File: bin/securedrop [hunk 1]
```diff
@@ -15,4 +16,23 @@ if __name__ == "__main__":
...
+        if o in ("-p", "--port"):
+            securedrop_port = a
+        elif o in ("-f", "--filename"):
+            securedrop_file = os.path.join(securedrop_dir, a)
+        elif o in ("-a", "--address"):
+            securedrop_hostname = a
+        elif o in ("-v", "--verbose"):
+            verbose_flag = True
+        else:
+            assert False, "unhandled option"
```
Comment (aokellermann) [inline bin/securedrop]: I'm not a big fan of asserts used like this. Assert is typically used for sanity checks, whereas this usage is more appropriate for some kind of exception. Please find an appropriate exception type to throw (when in doubt throw RuntimeError).

File: bin/securedrop [hunk 2]
```diff
@@ -15,4 +16,23 @@ if __name__ == "__main__":
...
+        print(err)  # will print something like "option -a not recognized"
+        sys.exit(2)
+    for o, a in opts:
+        if o in ("-p", "--port"):
+            securedrop_port = a
+        elif o in ("-f", "--filename"):
+            securedrop_file = os.path.join(securedrop_dir, a)
+        elif o in ("-a", "--address"):
+            securedrop_hostname = a
+        elif o in ("-v", "--verbose"):
```
Comment (aokellermann) [inline bin/securedrop:34]: Great idea to include this!

File: bin/securedrop_server [hunk 3]
```diff
@@ -15,4 +16,20 @@ if __name__ == "__main__":
...
+        sys.exit(2)
+    for o, a in opts:
+        if o in ("-p", "--port"):
+            securedrop_port = a
+        elif o in ("-f", "--filename"):
+            securedrop_file = a
+        elif o in ("-v", "--verbose"):
+            verbose_flag = True
+        else:
+            assert False, "unhandled option"
```
Comment (aokellermann) [inline bin/securedrop_server]: change assert here as well

**Review by aokellermann — CHANGES_REQUESTED** [review]
Nice work!

File: bin/securedrop (hunk 1 above)
Comment (BobbyBytes) [reply bin/securedrop]: OK, I couldn't find anything that seemed more appropriate than runtime error.

File: bin/securedrop (hunk 2 above)
Comment (BobbyBytes) [reply bin/securedrop:34]: TY

**Comment by BobbyBytes** [comment]
Assertions removed.

**Comment by jjjoycing123** [comment]
Looks good to me! No comment.

**Review by jjjoycing123 — APPROVED** [review]

**Review by aokellermann — APPROVED** [review]

**Pull request merged**